src/Plugins/ComplexCore/docs/FindVertexToTriangleDistances.md

```diff
@@ -0,0 +1,37 @@
+Find Vertex to Triangle Distances 
+=============
+
+## Group (Subgroup) ##
+Sampling (Spatial)
+
+## Description ##
+This **Filter** computes distances between points in a **Vertex Geoemtry** and triangles in a **Triangle Geoemtry**.  Specifically, for each point in the **Vertex Geometry**, the Euclidean distance to the closest triangle in the **Triangle Geoemtry** is stored.  This distance is *signed*: if the point lies on the side of the triangle to which the triangle normal points, then the distance is positive; otherwise, the distance is negative. Additionally, the ID the closest triangle is stored for each point.
```

Comment:
Did we ever figure out what is being used on the triangle to measure the distance? Is it the points of the triangle or the centroid? Please note this in the documentation.